api.planx.uk/hasura/voyager.ts

```diff
@@ -21,9 +21,12 @@ export default function graphQLVoyagerHandler(
     };
     if (adminKey) headers["x-hasura-admin-secret"] = adminKey;
 
-    const gqlUrl: string = process.env.HASURA_GRAPHQL_URL!;
+    const host: string = req.hostname;
+    const graphQLHost = host.replace(/^https:\/\/api/, "https://hasura");
+    const graphQLURL = `${graphQLHost}/v1/graphql`;
```

Comment:
We have the env vars `API_URL_EXT` and `EDITOR_URL_EXT` which are the external URLs of these services, but we've not hit the requirement for a `HASURA_URL_EXT` yet.

Adding that might be a good long term solution here 👍